components/director/internal/domain/application/repository.go

```diff
@@ -73,12 +73,12 @@ func (r *pgRepository) List(ctx context.Context, tenant string, filter []*labelf
 	if err != nil {
 		return nil, errors.Wrap(err, "while building filter query")
 	}
-	var additionalConditions string
+	var additionalConditions []string
```

Comment:
Earlier, in case when there were no additional conditions it was passing empty string as vararg parameter instead of no parameters at all, and there had to be an extra check in each helper making sure the condition wasn't an empty string. I believe it's clearer this way, also, I changed the way helpers build the query statements and keeping it the old way would require even more "hacks".

Reply:
After moving to system tenant, this change is no longer needed.